ONSdigital/takeon-ui
# Feature/spp 2146/UI validate ingestion

# Description
Story - https://collaborate2.ons.gov.uk/jira/browse/SPP-2146 - Tech: Update UI (override) to use standard BAW notification contract/queue

# How to test?
In the spp2548 environment:

1. Press 'Save and Validate' or 'Override' for a Contributor
2. Applicable only for 'Save and Validate' button - Check Cloudwatch log for spp-es-takeon-validation-spp2548-bpmtransform lambda to see if it successfully sent the message to SNS for 
3. Then check the actual message on the sns_message.fifo queue (this queue only for testing purposes)

For unit tests, run:
python3 -m pytest tests/ -vv --cov=. 

# Links
Jira: https://collaborate2.ons.gov.uk/jira/browse/SPP-2144
